<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>777</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MRS VISION SYSTEM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btok">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>680</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CONECTAR GIGe</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageIRlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>421</width>
      <height>321</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bFindCamera">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>PROCURAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_cameraname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>660</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SEM CAMERAS ENCONTRADAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="busbCam">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>670</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CONECTAR USB</string>
    </property>
   </widget>
   <widget class="QLabel" name="ImageUSBLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>421</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="bLoadFlirConfig">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>680</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CONFIGURAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageContourUSBLabel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imageCountourIRlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>421</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
